packages/components/CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 -   `withFocusReturn`: Refactor tests to `@testing-library/react` ([#45012](https://github.com/WordPress/gutenberg/pull/45012)).
 -   `ToolsPanel`: updated to satisfy `react/exhaustive-deps` eslint rule ([#45028](https://github.com/WordPress/gutenberg/pull/45028))
 -   `Tooltip`: updated to ignore `react/exhaustive-deps` eslint rule ([#45043](https://github.com/WordPress/gutenberg/pull/45043))
+-   `AnglePickerControl`: Set Storybook Label control type to 'text' ([#45122](https://github.com/WordPress/gutenberg/pull/45122)).
```

Comment:
We will need to move this to the `Unreleased` section of the changelog (this feedback applies to all currently open PRs)

Reply:
For this PR, done in https://github.com/WordPress/gutenberg/pull/45122/commits/dd2de2c48b7bfb6a59dc92792a978449a5bfae3a